docs/compatibility.md

```diff
@@ -104,21 +104,21 @@
 | Method | Supported |
 |--------|-----------|
 | allowsNull | :heavy_check_mark: Yes |
-| canBePassedByValue | todo |
-| getClass | todo |
+| canBePassedByValue | :heavy_check_mark: Yes |
+| getClass | :x: No - could currently cause recursion |
```

Comment:
We'd have to load a `ReflectionClass` instance of the class that is hinted in the language type hint, which could be the same class. Currently, because we are not autoloading classes, and the PR #3 is still WIP, we have no way of really loading a `ReflectionClass` for this, unless we used builtin `ReflectionClass`, but that kinda defeats the point. It's a mess that I need to have a think on...